movielens/requirements.txt

```diff
@@ -1,2 +1,3 @@
-tensorflow==1.2.0
-tensorflow-transform==0.1.10
+--index-url https://pypi.python.org/simple/
+
+-e .
```

Comment:
@puneith Not sure what this means? is it possible to add a comment on the requirements file?

Reply:
Ok added